doc/sphinx/addendum/micromega.rst

```diff
@@ -180,7 +180,7 @@ are a way to take into account the discreteness of :math:`\mathbb{Z}` by roundin
    Let :math:`p` be an integer and :math:`c` a rational constant. Then
    :math:`p \ge c \rightarrow p \ge \lceil{c}\rceil`.
 
-For instance, from 2 x = 1 we can deduce
+For instance, from :math:`2 x = 1` we can deduce
```

Comment:
```suggestion
For instance, from :math:`2x = 1` we can deduce
```

Isn't it more conventional without a space?

FWIW, I notice that later in the chapter there is this expression.  Seems to me it would be simpler to omit the multiplication symbols as in conventional in high school algebra--unless there's real reason to have them.  If there's a reason to have them in this expresion, should the multiply symbol only appear in the expression you changed?

![image](https://user-images.githubusercontent.com/1253341/91668643-14666a80-eac3-11ea-845a-282bbf9074e5.png)

Reply:
If `: math:` is like LaTeX, then it doesn't matter how many spaces there are.  Can you check if spaces matter?

And, yes, omitting the multiplication signs seems good, but I'm not at a computer right now (would you make that change?)

Reply:
> If `: math:` is like LaTeX, then it doesn't matter how many spaces there are.  Can you check if spaces matter?

You're right, it renders like this:

![image](https://user-images.githubusercontent.com/1253341/91670707-cc504380-ead4-11ea-96a5-7195dec443d7.png)

If you want to add the second fix in this PR, please do so.  Or I can merge as-is.

Reply:
Please merge as-is

Reply:
@coqbot: merge now